core/templates/dev/head/components/alerts/AlertMessageDirective.js

```diff
@@ -33,14 +33,20 @@ oppia.directive('alertMessage', [function() {
     ],
     link: function(scope) {
       var message = scope.getMessage();
+      var genericTimeout = 1500;
```

Comment:
There is no need of genericTimeout now, we can directly assign `message.timeout` to timeout. Since its going to be some value(which is not null or undefined).